app/assets/javascripts/application.js

```diff
@@ -87,6 +87,32 @@ function onContentLoad() {
         .find('.form_template')
         .remove();
     });
+
+  override_rails_confirm();
+}
+
+function override_rails_confirm() {
+  if (!$.rails || !tfm.tools) return;
+  $.rails.allowAction = function(element){
+    const message = element.data('confirm');
+    const isWarning = element.data('method') === 'delete';
+    if (!message) return true;
+
+    if ($.rails.fire(element, 'confirm')) {
+      tfm.tools.openConfirmModal({
```

Comment:
you can use our [legacy-js store](http://foreman.surge.sh/?path=/docs/introduction-legacy-js--page) and dispatch the action directly:
```js
tfm.store.dispatch(coreOpenConfirmModal(options))
```

Reply:
Thanks, moved to `foreman_overrides.js` and using it there :)